src/jmc/compile/datapack_data.py

```diff
@@ -6,6 +6,91 @@
 if TYPE_CHECKING:
     from ..compile.tokenizer import Token
 
+@dataclass
```

Comment:
Misuse of `@dataclass`

Reply:
I thought it is sth for storing data
thx for telling me that